execution_workflows/IsoSCM/bin/postprocess_identification.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+
+import sys
+import argparse
+import csv
+
+def parse_args(args=None):
+    Description = "Reformat IsoSCM bed file into the output file of identification challenge"
+    Epilog = "Example usage: python postprcoess_identification.py <FILE_IN> <IDENTIFICATION_OUT>"
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument("FILE_IN", help="Input IsoSCM output bed file.")
+    parser.add_argument("IDENTIFICATION_OUT", help="Name of output file for identification challenge")
+    return parser.parse_args(args)
+
+
+def reformat_bed(file_in, identification_out):
+    """
+    This function reformats IsoSCM output file to file for
+    identification challenge
+    :param file_in: IsoSCM assemble output bed file to be reformatted to identification bed file
+    :return: N/A
+    """
+    identification_out = open(identification_out, "wt")
+    with open(file_in) as fd:
+        rd = csv.reader(fd, delimiter="\t", quotechar='"')
+        for row in rd:
+            chrom = row[0]
+            start = row[3]
+            end = int(start) + 1
+            strand = row[6]
+            name = '|'.join([chrom, str(start)+":"+str(end), strand])
+            score = "."
+            output = (chrom, start, str(end), name, score, strand)
+            identification_out.write("\t".join(output) + "\n")
+
+    identification_out.close()
```

Comment:
Mr. One-off is back again :stuck_out_tongue:

The input file appears to be in GTF format (e.g. `results/isoscm/isoscm_assemble_outdir/control_r1/identification_out` & checking `modules/postprocess_identification.nf`):
![image](https://user-images.githubusercontent.com/49978382/161960805-2f1d901e-145b-4ec5-8ced-b86c954cd335.png)

GTF & BED files follow different conventions for reporting coordinates. GTF starts at 1, start and end are included included in range whilst BED starts at 0, start is included but end is excluded from the range. To convert GTF/GFF -> BED, you just need to subtract 1 from the start coordinate. 

P.S. I always like to refer to this [cheat sheet](https://gist.github.com/ilevantis/6d6ecf8718a5803acff736c2dffc933e#gff-vs-bed-indexing) for things like this as it has a nice visualisation (and I have to sanity check these things every time).

Reply:
hahahahaha ahhh good catch! thank you so much @SamBryce-Smith!! :D also thank you for sharing the cheat sheet!